examples/source_root/expected.js.map.golden

```diff
@@ -1,6 +1,6 @@
 {
     "sources": [
-        "in.ts"
+        "examples/source_root/in.ts"
```

Comment:
is this going to be inconsistent? Shouldn't ALL paths in the sources here become repo-relative?

Reply:
You mean throughout all test cases?

By default these `sources` are relative to the .map file, so no path navigation is needed when the .ts and .map are side-by-side. The exception being when an alternative root is specified (this fix), or when `root_dir|out_dir` are used to modify the path then we need to navigate there.

Reply:
The build rule that generated this was https://github.com/aspect-build/rules_swc/blob/8864f85e257078ae5e04739ffab6f6a356f2f237/examples/source_root/BUILD.bazel#L21 right?
I don't see how the relative location for the expected.js needs an extra examples/source_root to navigate back to the in.ts file in this case. It seems like the logic has to be updated to read the value passed to the source_root

Reply:
I think either one actually works when there is 1 .map for each .js. The "examples/source_root" path needs to live in one of the two so when you append them you get "examples/source_root/in.ts". But sourcemaps were designed to work with 1 .map for N .js and I think we should align with how that works, in which case the `sources` needs that path in it.